packages/cli/src/index.ts

```diff
@@ -0,0 +1,20 @@
+import { Command } from 'commander';
+import { createPieceCommand } from './lib/commands/create-piece';
+
+const program = new Command();
+
+program.version('0.0.1').description('Activepieces CLI');
+
+program
+  .command('create-piece')
```

Comment:
Please check https://www.npmjs.com/package/inquirer

I think it can helps with commanders to make the cli  interactive